sql/firefox_accounts_derived/fxa_users_services_first_seen_v1/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: FxA Users Services First Seen
+description: Usage aggregations describing when each FxA user was first seen
+owners:
+  - jklukas@mozilla.com
+labels:
+  application: fxa
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_fxa_events
+  # We completely recreate this table every day;
+  # making it incremental is possible but nuanced since it windows over
+  # events that may cross the midnight boundary.
+  date_partition_parameter: null
```

Comment:
We now sidestep the destination problem by copying the `init.sql` logic into `query.sql` so the execution method is more in line with other queries. This results in code duplication, but that's the case to some extent with existing practice for other `init.sql` queries.